autofill/autofill-impl/src/main/res/values/autofill-impl-strings-untranslated.xml

```diff
@@ -24,13 +24,17 @@
     <string name="managementDisabledModeTitle">Secure your device to use Autofill</string>
 
     <string name="saveLoginDialogTitle">Save Login?</string>
+    <string name="saveLoginMissingUsernameDialogTitle">Save Password?</string>
     <string name="saveLoginDialogButtonSave">Save Login</string>
+    <string name="saveLoginMissingUsernameDialogButtonSave">Save Password</string>
     <string name="saveLoginDialogNotNow">Not Now</string>
     <string name="saveLoginDialogOnboardingExplanationSubtitle">Logins are stored securely on this device only, and can be managed from the Autofill menu in Settings.</string>
-    <string name="saveLoginDialogFirstTimeExplanationTitle">Do you want DuckDuckGo\nto save your Login?</string>
+    <string name="saveLoginDialogFirstTimeOnboardingExplanationTitle">Do you want DuckDuckGo\nto save your Login?</string>
```

Comment:
NIT: This wouldn't yield a necessarily good result on bigger fonts. Why do we want to force a line break here instead of letting the system handle it?

Reply:
Good point. I was using the line break as a shortcut to achieve a balanced layout of the words. What I actually wanted was to use `android:breakStrategy="balanced"`.